devlib/module/cpufreq.py

```diff
@@ -375,9 +375,18 @@ def set_all_governors(self, governor):
         """
         Set the specified governor for all the (online) CPUs
         """
-        return self.target._execute_util(
+        try:
+            return self.target._execute_util(
                 'cpufreq_set_all_governors {}'.format(governor),
                 as_root=True)
+        except TargetError:
```

Comment:
True, but there isn't anything useful in the error output that tells you that unsupported governors is the cause. You could check for "sh: echo: I/O error" (which is what gets printed by shutils) but I think that could be caused by other stuff too, if not by someone yanking the cable..